nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/random/custom/RandomGamma.java

```diff
@@ -0,0 +1,62 @@
+package org.nd4j.linalg.api.ops.random.custom;
```

Comment:
Copyright header.
Also for Poisson and Shuffle ops.

Reply:
Fixed.